src/vmm/src/cpu_config/x86_64/static_cpu_templates/mod.rs

```diff
@@ -50,3 +50,40 @@ impl StaticCpuTemplate {
         self == &StaticCpuTemplate::None
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use std::path::PathBuf;
+
+    use super::*;
+    use crate::cpu_config::templates::CustomCpuTemplate;
+
+    // Get a static CPU template stored as a JSON file.
+    fn get_json_template(filename: &str) -> CustomCpuTemplate {
```

Comment:
This piece seems to be very similar between x86_64 and aarch64. Would it be too much to try to extract it?

Reply:
you're right. will try to define this in common test utilities.

Reply:
done